index.html

```diff
@@ -6,6 +6,6 @@ <h1>AllanJavaScript</h1>
 <script src="loops.js"></script>
 <script src="challenge5.js"></script>
 
-<img src="challenge5a.png" alt="Challenge 5">
+<img src="challenge5a.png" alt="Challenge 5"><br>
 <img src="challenge5b.png" alt="Challenge 5">
```

Comment:
use styles instead of <br> can CSS be used?

```
.block-image {
	display: block;
}
```

Best to use CSS for layouts as this will make building a responsive UI easier.

Reply:
I love that you reviewed the random html image-dump page too :P .. ok .. I will take your comments on board and make that nicer.